### PR TITLE
doc: Add guidelines for contacting Support DOCS-394

### PR DESCRIPTION
**Pull request by prcr**
Adds a checklist of the information necessary to troubleshoot the coverage setup and that users should include when contacting Support to make the process more efficient.

### :eyes: Live preview
https://feature-update-codacy-coverage-repo--docs-codacy.netlify.app/coverage-reporter/#validating-coverage

### CODE REVIEW

**Review by madalenamelo — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -302,5 +302,15 @@ To validate that the coverage setup is complete:
...
+
+    !!! note "Opening a Support ticket"
+        If you need more help setting up coverage on your repository please contact us at <mailto:support@codacy.com> including the following information:
+
+        -   URL of your repository on Codacy
+        -   Your CI/CD configuration files and the name of your CI/CD platform
+        -   Full console output of your CI/CD when running the Codacy Coverage Reporter
+        -   Branch name and commit UUID corresponding to the CI/CD output
+        -   Test coverage report that you're uploading to Codacy
+        -   Any other relevant information or screenshots of your setup
```
Comment (pedrocodacy) [inline docs/index.md:316]: this is sooooo nice :)

Comment (prcr) [reply docs/index.md:316]: Thanks, @pedrocodacy! Let me know if there is some other information that users could provide that would help us troubleshoot their setup. :+1:

Comment (IGFCoimbra) [reply docs/index.md:316]: This is indeed amazing :D

**Review by IGFCoimbra — APPROVED** [review]

**Comment by prcr** [comment]
I'm going to deploy these changes now, and we'll check if this is helping during the next coming weeks. :rocket:

**Pull request merged**